app/src/main/java/org/simple/clinic/storage/Migration_17_18.kt

```diff
@@ -0,0 +1,72 @@
+package org.simple.clinic.storage
+
+import android.arch.persistence.db.SupportSQLiteDatabase
+import android.arch.persistence.room.migration.Migration
+import org.simple.clinic.bp.BloodPressureMeasurement
+import org.simple.clinic.drugs.PrescribedDrug
+
+/**
+ * Removes foreign key references to Facility ID from [BloodPressureMeasurement] and [PrescribedDrug].
+ */
+@Suppress("ClassName")
+class Migration_17_18 : Migration(17, 18) {
+
+  override fun migrate(database: SupportSQLiteDatabase) {
+    migrateBloodPressureMeasurement(database)
+    migratePrescribedDrug(database)
+  }
+
+  private fun migrateBloodPressureMeasurement(database: SupportSQLiteDatabase) {
+    // Sqlite doesn't support dropping constraints, so we recreate the table.
+    database.execSQL("ALTER TABLE `BloodPressureMeasurement` RENAME TO `BloodPressureMeasurement_v17`")
```

Comment:
Can you please use double quotes instead of backticks in SQL statements?

Reply:
Same

> Since this class has usages of both backticks and double quotes, with backticks being the majority, let's keep this for now until we decide on one standard. I'll add this to my discuss-once-pratul-is-back TODOs.

Reply:
doing this